src/parser/jobs/mnk/modules/Fists.js

```diff
@@ -185,38 +187,47 @@ export default class Fists extends Module {
 
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.FISTS_OF_FIRE.icon,
-			content: <Fragment>
-				Fist buffs are one of your biggest DPS contributors, either directly with <ActionLink {...ACTIONS.FISTS_OF_FIRE} /> or <StatusLink {...STATUSES.GREASED_LIGHTNING_I} /> manipulation with <ActionLink {...ACTIONS.FISTS_OF_EARTH} /> and <ActionLink {...ACTIONS.FISTS_OF_WIND} />.
-			</Fragment>,
-			why: `${this._fistGCDs[STANCELESS]} GCDs had no Fists buff active.`,
+			content: <>
+				<Trans id="mnk.fists.suggestions.stanceless.content">Fist buffs are one of your biggest DPS contributors, either directly with <ActionLink {...ACTIONS.FISTS_OF_FIRE} /> or <StatusLink {...STATUSES.GREASED_LIGHTNING_I} /> manipulation with <ActionLink {...ACTIONS.FISTS_OF_EARTH} /> and <ActionLink {...ACTIONS.FISTS_OF_WIND} />.</Trans>
+			</>,
+			why: <>
```

Comment:
Unnecessary fragment, and you could probably move most of that string out of the plural.